test/public/dataPasses/overviewPerLhcPeriod.test.js

```diff
@@ -67,6 +69,41 @@ module.exports = () => {
         };
 
         await validateTableData(page, new Map(Object.entries(tableDataValidators)));
+        const names = await getAllDataFields(page, 'name');
+        const runs = await getAllDataFields(page, 'associatedRuns');
+        const anchoreds = await getAllDataFields(page, 'anchoredSimulationPasses');
+        expect(names.map((name, index) => ({
+            name,
+            runsCount: Number(runs[index].split('\n')[0]) || 0,
+            simulationPassesCount: Number(anchoreds[index].split('\n')[0]) || 0,
+        }))).to.have.all.deep.members([
```

Comment:
This is a very far-fetched solution. It works, but getting data column by column, then build rows, and finally check it is hard to understand.
Can't you do `$eval` on rows, then inside it build an object returning the `{name, runsCount, simulationPassesCount}` using innerText?

Reply:
refactored